addon/tests/helpers/with-feature.js

```diff
@@ -1,8 +1,8 @@
 import Ember from 'ember';
+import features from 'ember-feature-flags/features';
 
 export function withFeature( featureName ){
-  window.Features = window.Features || {};
-  window.Features[featureName] = true;
+  features.set(featureName, true);
```

Comment:
So I think the approach here should be as follows, this is what I'll be submitting in this PR:

- Stick with the module approach, the container service approach is too fiddly
- In tests that call `startApp()` the initial feature set will be loaded from `config/environment.js`, but you can easily override by passing `FEATURES` in the `attrs` array:

    ```javascript
    startApp({FEATURES: {foo: true}})
    ```
- In unit tests (that don't call `startApp()`) `resetFeatureFlags()` and `withFeature()` can be used to manage the feature set. Care must be taken to call `resetFeatureFlags()` though because the feature set persists between tests
- It should be possible to bind to a feature flag: `{{#if features.flags.foo}}...`

This won't be included in my PR unless you agree, but I think to make the bound form more readable I think we should:

- Change the injection to be `this.feature` (this reads well: `if (this.feature.enabled('foo')) { ...`)
- Consider using `unknownProperty()` to that you can have `{{#if feature.foo}}` - @kategengler what do you think about this idea? I've used this approach for ACL checks before and it works well (`{{#if acl.create}}...`)